packages/fleet/lib/fleet.integration.test.ts

```diff
@@ -0,0 +1,29 @@
+import { expect, describe, it, beforeAll, afterAll, afterEach } from 'vitest';
+import { Fleet } from './fleet.js';
+import { getTestDbClient, testDbUrl } from './db/helpers.test.js';
+import { generateCommitHash } from './models/helpers.test.js';
+
+describe('fleet', () => {
+    const fleetId = 'my_fleet';
+    const fleet = new Fleet({ fleetId, dbUrl: testDbUrl });
+
+    beforeAll(async () => {
+        await fleet.migrate();
+    });
+
+    afterEach(() => {});
+
+    afterAll(async () => {
+        await getTestDbClient(fleetId).clearDatabase();
+    });
+
+    describe('deploy', () => {
+        it('should create a new deployment', async () => {
```

Comment:
You are right. On the other hand stub everywhere could introduce more issues down the line when underlying logic is changed. 

I've added this test to bootstrap the fleet.integration.test file but I might remove it since it is not very useful